app/services/albums.js

```diff
@@ -26,3 +27,18 @@ exports.getAlbum = async id => {
     throw errors.albumApiError(err.message);
   }
 };
+
+exports.getAlbums = async (offset, limit, orderBy) => {
+  const url = `${ALBUM_API_URL}/albums`;
+  logger.info(`Making a request to url ${url}`);
+  try {
+    const albums = await axios.get(`${url}`);
+    const paginationAlbums = albums.data.slice(offset, offset + limit);
```

Comment:
Im torn here, my train of thought would be to have this logic in the resolver rather than here and keep the service function as "pure" as posible, meaning, returning exactly what the service is supposed to return since the pagination you implemented is more a concert of the graphql service rather than the album api.
Any thoughts?